src/apps/Permissions/Home/Apps.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { CardLayout } from '@aragon/ui'
+import { AppType } from '../../../prop-types'
+import AppCard from '../AppCard'
+import EmptyBlock from '../EmptyBlock'
+
+function Apps({ apps, loading, onOpenApp }) {
+  if (loading) {
+    return <EmptyBlock>Loading apps…</EmptyBlock>
+  }
+
+  if (apps.length === 0) {
+    return <EmptyBlock>No apps installed.</EmptyBlock>
+  }
+
+  return (
+    <CardLayout rowHeight={170}>
```

Comment:
❤️👍 this makes so much sense, had not imagined we can use it this way.

Reply:
Yes! And `Card` stops having explicit dimensions when used in `CardLayout`: https://github.com/aragon/aragon-ui/blob/newstyle/src/components/Card/Card.js#L13-L15